src/main/java/teammates/ui/controller/FeedbackSubmissionEditPageData.java

```diff
@@ -37,6 +42,10 @@ public FeedbackSubmissionEditPageData(AccountAttributes account, StudentAttribut
         isShowRealQuestionNumber = false;
         isHeaderHidden = false;        
     }
+    
+    public void init() {
```

Comment:
The function name is not very informative. Add comments for this? And if this is really done for initialization, why is it not in the constructor?